test/bddtests/features/ledgerconfig.feature

```diff
@@ -10,6 +10,7 @@ Feature: ledger-config
 
   Background: Setup
     Given the channel "mychannel" is created and all peers have joined
+    And "test" chaincode "configscc" is instantiated from path "in-process" on the "mychannel" channel with args "" with endorsement policy "AND('Org1MSP.member','Org2MSP.member')" with collection policy ""
```

Comment:
if I recall the previous PR, it's called `configcc` now (ie not `confiscc`), does this apply here?

Reply:
I think it has always been configscc. Maybe configcc is more appropriate. Possibly in a future change.

Reply:
my bad, that's in a different project.. never mind my previous comment then.